src/desktop.md

```diff
@@ -5,46 +5,58 @@ toc: true
 ---
 
 Desktop support allows you to compile Flutter source code
-to a native Windows, macOS, or Linux desktop app. Flutter's desktop
-support also extends to plugins&mdash;you can install 
-existing plugins that support the Windows, macOS, or Linux platforms,
-or you can create your own.
+to a native Windows, macOS, or Linux desktop app.
+Flutter's desktop support also extends to plugins&mdash;you
+can install existing plugins that support the Windows,
+macOS, or Linux platforms, or you can create your own.
 
 {{site.alert.warning}}
   **Beta!**
   This page covers desktop support,
-  which is available as beta-quality
-  features in the Flutter beta channel.
-  Support still has notable feature gaps,
+  which is available as a beta release.
+  Beta support still has notable feature gaps,
   including accessibility support.
+  You can try a beta snapshot of desktop support
+  on the stable channel, or you can keep up
+  with the latest changes to desktop on the
+  beta channel.
+  For more information, see the **Desktop**
+  section in [What's new in Flutter 2][],
+  a free article on Medium.
 {{site.alert.end}}
 
+[What's new in Flutter 2]: {{site.flutter-medium}}/whats-new-in-flutter-2-0-fe8e95ecc65
+
+
 {{site.alert.note}}
   To compile a desktop app, you must build the app **on**
   the targeted platform: build a Windows app on Windows,
   a macOS app on macOS, and a Linux app on Linux.
   If you experience a problem that hasn’t yet been reported,
-  please file an issue and include
+  please [file an issue][] and include
   "desktop:macos/linux/windows"
   (whichever platform is appropriate) in the title.
 {{site.alert.end}}
 
-## Beta Snapshot in Stable channel
+## Beta Snapshot in stable channel
 
-To make it easier to try out Desktop support for Flutter, we are
-shipping a snapshot of Flutter beta channel’s Desktop support in
-Flutter 2 stable. This means that you can easily try it out without
-needing to switch to the Flutter beta channel. However, the snapshot
-included in the stable channel will not be updated with the latest
-Flutter support for Desktop until the next Flutter stable release.
+To make it easier to try out desktop support for
+Flutter, we are shipping a snapshot of Flutter
+beta channel’s desktop support in Flutter 2 stable.
```

Comment:
I would word this as: "we are shipping a snapshot of Flutter's desktop beta on the stable channel for Flutter 2."